services/core-api/app/api/mines/permits/permit/resources/permit.py

```diff
@@ -23,55 +23,42 @@ class PermitListResource(Resource, UserMixin):
     parser.add_argument(
```

Comment:
This file is all formatting changes.. 

my settings.json is the same as the mds/settings.json. Not sure why this is happening

```
  "python.formatting.yapfArgs": [
    "--style={column_limit:100, spaces_before_comment:'15,30,50', SPLIT_BEFORE_EXPRESSION_AFTER_OPENING_PAREN:TRUE}"
  ],
```